### PR TITLE
Add description view files

### PR DESCRIPTION
**Pull request by fmauch**
This MR adds launch files for viewing the individual descriptions with zero overhead.

### CODE REVIEW

**Review by miguelprada — DISMISSED** [review]
Thanks @fmauch!

File: ur_description/launch/view_ur10.launch [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<launch>
+  <include file="$(find ur_description)/launch/ur10_upload.launch"/>
+
+  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" >
+    <param name="use_gui" value="true"/>
+  </node>
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
+  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find ur_e_description)/etc/view_robot.rviz" required="true" />
```
Comment (gavanderhoorn) [inline ur_description/launch/view_ur10.launch]: this is a `.launch` file in `ur_description`, but it uses the RViz config from `ur_e_description`, even though `ur_description` also has an RViz config now.

Was that intended?

Comment (gavanderhoorn) [reply ur_description/launch/view_ur10.launch]: Additionally: could we place the RViz config in a `cfg` sub dir? Instead of the `etc`?

**Review by gavanderhoorn — CHANGES_REQUESTED** [review]
If you could address the two small comments this could be merged.

Nice contribution :+1:

File: ur_description/launch/view_ur10.launch (hunk 1 above)
Comment (fmauch) [reply ur_description/launch/view_ur10.launch]: I changed the files accordingly

**Review by gavanderhoorn — APPROVED** [review]
Thanks again for the PR :+1:

**Review by gavanderhoorn — APPROVED** [review]
Thanks again :+1: 

looks like travis finally caught up.

**Comment by gavanderhoorn** [comment]
Thanks @fmauch.

:+1: :medal_sports:

**Pull request merged**